src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -72,7 +71,7 @@ function IOURequestStepDescription({
     );
 
     const navigateBack = () => {
-        Navigation.goBack(backTo || ROUTES.HOME);
```

Comment:
I thinkkkk getting this piece to staging will fix these blockers:
- https://github.com/Expensify/App/issues/36306
- https://github.com/Expensify/App/issues/36307

Reply:
CP'd just this to staging yesterday to unblock deploy, in https://github.com/Expensify/App/pull/36329